NEWS.md

```diff
@@ -1,7 +1,7 @@
 # usethis (development version)
 
-* usethis will now overwrite files without prompting if you are using git
-  because you can undo if it's a problematic change (#1376).
+* usethis will now overwrite files without prompting if you set
+  `option(usethis.auto_write = TRUE)` and are using git (#1376).
```

Comment:
Other ideas: `usethis.auto_overwrite` or `usethis.auto_write_over`.

Reply:
`usethis.safety = "off"` ?

Reply:
I prefer the specificity of the other alternatives. `usethis.safety = "off"` would need more explanation. Of course, all of them would need to be documented, but I think the need to re-read those docs would be much higher with `usethis.safety`.

Reply:
What about `usethis.overwrite = TRUE`, that seems relatively self documenting to me.

Reply:
Yeah, I like that.